interfaces/builtin/avahi_observe.go

```diff
@@ -19,14 +19,212 @@
+
+dbus (send)
+    bus=system
+    interface=org.freedesktop.Avahi.AddressResolver
+    peer=(name=org.freedesktop.Avahi, label=###PLUG_SECURITY_TAGS###),
+
+# host name resolving
+dbus (receive)
+    bus=system
+    path=/Client*/HostNameResolver*
+    interface=org.freedesktop.Avahi.HostNameResolver
+    peer=(label=###PLUG_SECURITY_TAGS###),
+
+dbus (send)
+    bus=system
+    interface=org.freedesktop.Avahi.HostNameResolver
+    peer=(name=org.freedesktop.Avahi, label=###PLUG_SECURITY_TAGS###),
+
+# service resolving
+dbus (receive)
+    bus=system
+    path=/Client*/ServiceResolver*
+    interface=org.freedesktop.Avahi.ServiceResolver
+    peer=(label=###PLUG_SECURITY_TAGS###),
+
+dbus (send)
+    bus=system
+    interface=org.freedesktop.Avahi.ServiceResolver
+    peer=(name=org.freedesktop.Avahi, label=###PLUG_SECURITY_TAGS###),
+
+# domain browsing
+dbus (receive)
+    bus=system
+    path=/Client*/DomainBrowser*
+    interface=org.freedesktop.Avahi.DomainBrowser
+    peer=(label=###PLUG_SECURITY_TAGS###),
+
+dbus (send)
+    bus=system
+    interface=org.freedesktop.Avahi.DomainBrowser
+    peer=(name=org.freedesktop.Avahi, label=###PLUG_SECURITY_TAGS###),
+
+# record browsing
+dbus (receive)
+    bus=system
+    path=/Client*/RecordBrowser*
+    interface=org.freedesktop.Avahi.RecordBrowser
+    peer=(label=###PLUG_SECURITY_TAGS###),
+
+dbus (send)
+    bus=system
+    interface=org.freedesktop.Avahi.RecordBrowser
+    peer=(name=org.freedesktop.Avahi, label=###PLUG_SECURITY_TAGS###),
+
+# service browsing
+dbus (receive)
+    bus=system
+    path=/Client*/ServiceBrowser*
+    interface=org.freedesktop.Avahi.ServiceBrowser
+    peer=(label=###PLUG_SECURITY_TAGS###),
+
+dbus (send)
+    bus=system
+    interface=org.freedesktop.Avahi.ServiceBrowser
+    peer=(name=org.freedesktop.Avahi, label=###PLUG_SECURITY_TAGS###),
+
+# service type browsing
+dbus (receive)
+    bus=system
+    path=/Client*/ServiceTypeBrowser*
+    interface=org.freedesktop.Avahi.ServiceTypeBrowser
+    peer=(label=###PLUG_SECURITY_TAGS###),
+
+dbus (send)
+    bus=system
+    interface=org.freedesktop.Avahi.ServiceTypeBrowser
+    peer=(name=org.freedesktop.Avahi, label=###PLUG_SECURITY_TAGS###),
+`
+
+const avahiObservePermanentSlotDBus = `
+<!-- Only root can own the Avahi service -->
+<policy user="root">
+  <allow own="org.freedesktop.Avahi"/>
+</policy>
+
+<!-- Allow anyone to invoke methods on Avahi server, except SetHostName -->
+<policy context="default">
+  <allow send_destination="org.freedesktop.Avahi"/>
+  <allow receive_sender="org.freedesktop.Avahi"/>
+
+  <deny send_destination="org.freedesktop.Avahi"
+        send_interface="org.freedesktop.Avahi.Server" send_member="SetHostName"/>
+</policy>
+
+<!-- Allow everything, including access to SetHostName to users of the group "netdev" -->
+<policy group="netdev">
+  <allow send_destination="org.freedesktop.Avahi"/>
+  <allow receive_sender="org.freedesktop.Avahi"/>
+</policy>
```

Comment:
Good find, indeed makes no sense here.
I removed the policy and added comment why so.